lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -96,8 +96,16 @@ def configure(instance):
         type=int,
         help='Number of instances'
     )
+    @click.option(
+        '--disk',
+        required=True,
+        default=10,
+        type=int,
```

Comment:
put a todo to convert it to qualified string. style failed anyways, so you need to make a change.

Reply:
Why does it need to be converted to string? boto3 expects an integer value.

Reply:
i find --size 10 confusing from ux perspective. --disk-size 10GB is easier to read. so - not important, not now if ever.

Reply:
That makes sense- I changed the CLI option and added a callback to convert it to int.